__tests__/endpoints.test.js

```diff
@@ -106,6 +106,43 @@ describe("/api/articles", () => {
   });
 });
 
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200 respond with all the comments associated to the given article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        for (const comment of body.comments) {
+          expect(comment.article_id).toBe(1);
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            author: expect.any(String),
+          });
+        }
+      });
+  });
+  test("GET: 400 responds with an appropriate status and error message when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/not-a-number/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("GET: 404 responds with an appropriate status and error message when given a non-existent id", () => {
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+});
```

Comment:
Good work so far, but we have one untested behaviour here:
We have valid articles which have no associated comments. In this case we would expect status 200 responds with empty array of comments